pallets/author-mapping/src/lib.rs

```diff
@@ -119,19 +142,67 @@ pub mod pallet {
 		/// Users who have been (or will soon be) elected active collators in staking,
 		/// should submit this extrinsic to have their blocks accepted and earn rewards.
 		#[pallet::weight(<T as Config>::WeightInfo::add_association())]
-		pub fn add_association(origin: OriginFor<T>, author_id: NimbusId) -> DispatchResult {
+		pub fn add_association(
```

Comment:
For compatibility reason, we need to keep the old extrinsic, create a new one instead.

The old extrinsic should set the VRF key as the same value as the provided `NimbusId`.

Reply:
I created the new extrinsic already called set_keys below. I agree we need to keep the old one for compatibility reasons.

I'll make the change to add_association so it sets the VRF key to the new NimbusId.